util/intstr/parse.go

```diff
@@ -0,0 +1,9 @@
+package intstr
+
+import "k8s.io/apimachinery/pkg/util/intstr"
+
+// convenience func to get a pointer
+func Parse(val string) *intstr.IntOrString {
```

Comment:
```suggestion
func ParsePointer(val string) *intstr.IntOrString {
```
?

Reply:
I've made it `ParsePtr` as this is common